index.html

```diff
@@ -149,6 +153,27 @@ <h4>src<a href="#required-for-display">*</a></h4>
             <h4>alt</h4>
             <p>Configures the model with custom text that will be used to describe the model to viewers who use a screen reader or otherwise depend on additional semantic context to understand what they are viewing.</p>
           </li>
+          <li>
+            <h4>animated</h4>
```

Comment:
I think `animate` might be a better attribute name than `animated` (present vs past tense), although can't think of good prior art (other than "checked", which is past tense, but seems like a different case?)

Reply:
Will fix

Reply:
Ugh, I forgot: I originally used `animate`, but it's a built-in method of HTMLElement (part of the Web Animations API).

Reply:
(and yah that's definitely not going to be confusing for folks _at all_)

😩

Reply:
This has been left as is, but if there is another name we like that doesn't overlap with a built-in API, I am open to changing it.